src/AiSystem.h

```diff
@@ -9,6 +13,23 @@ class AiSystem : public EventSystem<AiSystem> {
 
 private:
     void handle_activate_ai(const Event& e);
+    void handle_object_transform(const Event& e);
+
+    void path_to(int car, const glm::vec3& point_);
+
     int num_ai_;
+
+    struct car_data {
+        struct car_transform_data {
```

Comment:
Isn't this struct unnecessary?

Just `car_transform_data.rotation_` should suffice right?

Reply:
We might want to add more to this struct in the future, like what type of power-up they have